references/classification/train.py

```diff
@@ -130,8 +130,8 @@ def load_data(traindir, valdir, args):
     else:
         auto_augment_policy = getattr(args, "auto_augment", None)
         random_erase_prob = getattr(args, "random_erase", 0.0)
-        ra_magnitude = args.ra_magnitude
-        augmix_severity = args.augmix_severity
+        ra_magnitude = getattr(args, "ra_magnitude", None)
+        augmix_severity = getattr(args, "augmix_severity", None)
```

Comment:
Instead of `None`, shouldn't we use the correct defaults here?

https://github.com/pytorch/vision/blob/7cf0f4cc1801ff1892007c7a11f7c35d8dfb7fd0/references/classification/train.py#L462-L463

https://github.com/pytorch/vision/blob/7cf0f4cc1801ff1892007c7a11f7c35d8dfb7fd0/references/classification/presets.py#L16-L17

```suggestion
        ra_magnitude = getattr(args, "ra_magnitude", 9)
        augmix_severity = getattr(args, "augmix_severity", 3)
```

There should be no harm in doing not so, because `assert auto_augment_policy is None` for the quantized training and thus we'll never actually use the value. Still, `random_erase_prob` also has the "correct" default.

Reply:
@datumbox

> If you are certain RA/Augmix can't be reached in the quantization branch,

They can't:

https://github.com/pytorch/vision/blob/7cf0f4cc1801ff1892007c7a11f7c35d8dfb7fd0/references/classification/presets.py#L23-L32

https://github.com/pytorch/vision/blob/7cf0f4cc1801ff1892007c7a11f7c35d8dfb7fd0/references/classification/train.py#L131-L140

As long as `args.auto_augment_policy` is `None`, we'll never touch any of these transformations. Since the quantization script doesn't define it, we will always get that and should be safe.

Reply:
@NicolasHug 

I'm ok with leaving them `None`. Should we make it more obvious that we are just filling in blanks from the quantization script? Maybe add a comment there?